files/ja/learn/getting_started_with_the_web/how_the_web_works/index.html

```diff
@@ -93,7 +93,7 @@ <h2 id="DNS_explained">DNS の説明</h2>
 
 <p>これは、 {{Glossary("IP Address", "IP アドレス")}}と呼ばれ、ウェブ上の一意の場所を表します。しかし、あまり覚えやすくはないですよね。だから、ドメインネームサーバーが発明されたのです。これらは特別なサーバーで、ブラウザーに入力したウェブアドレス ("mozilla.org" など) とウェブサイトの実際の (IP) アドレスを対応させます。</p>
 
-<p>ウェブサイトには、IP アドレスを使用して直接アクセスできます。<a href="https://ipinfo.info/html/ip_checker.php">IP Checker</a> のようなツールにドメインを打ち込むことで、ウェブサイトの IP アドレスを見つけることができます。</p>
+<p>ウェブサイトには、IP アドレスを使用して直接アクセスできます。<a href="https://www.nslookup.io/website-to-ip-lookup/">IP Checker</a> のようなツールにドメインを打ち込むことで、ウェブサイトの IP アドレスを見つけることができます。</p>
```

Comment:
Can you replace the link text?

```suggestion
<p>ウェブサイトには、IP アドレスを使用して直接アクセスできます。<a href="https://www.nslookup.io/website-to-ip-lookup/">DNS lookup</a> のようなツールにドメインを打ち込むことで、ウェブサイトの IP アドレスを見つけることができます。</p>
```
```suggestion
<p>ウェブサイトには、IP アドレスを使用して直接アクセスできます。<a href="https://www.nslookup.io/website-to-ip-lookup/">IP Checker</a> のようなツールにドメインを打ち込むことで、ウェブサイトの IP アドレスを見つけることができます。</p>
```